client/my-sites/site-settings/section-import.jsx

```diff
@@ -69,21 +66,25 @@ export default React.createClass( {
 		}
 
 		return status.map( item => Object.assign( {}, item, { site, siteTitle } ) );
-	},
+	}
 
-	updateFromAPI: function() {
+	updateFromAPI = () => {
 		fetchState( this.props.site.ID );
-	},
+	};
 
-	updateState: function() {
+	updateState = () => {
 		this.setState( getImporterState() );
-	},
+	};
```

Comment:
Also removed in https://github.com/Automattic/wp-calypso/pull/13182/commits/ec3a280c8f6199cd8c5711c2fbf1ad8f15ee80dd.